### DISCUSSION TITLE
Unable to read exodus file

### DISCUSSION THREAD

**Discussion by HerambMahajan** [idaholab/moose#20372] [Q&A Meshing] [answered]
I am using Cubit to mesh a geometry and import exodus '.e' file in MOOSE. But I am getting following error:

*** ERROR ***
The following error occurred in the object "mesh", of type "FileMesh".

Unable to nodeset ID: -123.

Am I missing something during meshing or importing file in MOOSE?

**Comment by GiudGiud** [comment, marked as answer]
Hello

I havent seen this error before. Can you share your mesh file? Either here or by email at my address listed on github

Guillaume

**Reply by GiudGiud** [reply]
Also you should try to use this instead of specifying the mesh directly in the [Mesh] block:
https://mooseframework.inl.gov/source/meshgenerators/FileMeshGenerator.html
